sky/shell/platform/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -315,8 +315,9 @@ - (BOOL)accessibilityPerformMagicTap {
     }
   }
 
-  if (!view_.accessibilityElements) {
-    view_.accessibilityElements = @[ objects_[kRootNodeId] ];
+  SemanticObject* root = objects_[kRootNodeId];
```

Comment:
Under what conditions do we receive an initial update (i.e. not having set view_.accessibilityElements yet) not containing root?